explore/src/main/scala/explore/targeteditor/SiderealTargetEditor.scala

```diff
@@ -267,7 +267,9 @@ object SiderealTargetEditor {
                 props.uid,
                 props.id,
                 props.posAngle,
-                props.vizTime,
+                // This may need to go into a effect but it complicates the code quite a bit, and
+                // we need precission of months/years
```

Comment:
You can `useEffectResult` and then `potRender` (or just `.toOption`).

Reply:
Excellent tip. Updated